projects/plugins/jetpack/extensions/shared/icons.js

```diff
@@ -237,6 +237,44 @@ export const JetpackLogo = ( { size = 24, border = 0, className, color = COLOR_J
 	);
 };
 
+export const JetpackMobileAppIcon = () => {
```

Comment:
My initial reaction with those is that it's 2 more icons we'll have to keep updated. I was wondering if we couldn't just use the existing regular Jetpack icon, but I suppose that wouldn't fit with the designs you had in mind.

Reply:
Yeah the Jetpack icon is not the same as the design.